README.md

```diff
@@ -1188,6 +1191,43 @@ All options from BDD tests do also apply here.
 
 **NOTE:** Performance tests should be run without concurrency.
 
+#### List of test tags
+
+| Tag name           | Tag meaning                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------- |
+| @smoke             | Smoke tests verifying basic functionality                                          |
+| @performance       | Performance tests                                                                  |
+| @olm               | OLM integration tests                                                              |
+| @travelagency      | Travel agency tests                                                                |
+|                    |                                                                                    |
+| @disabled          | Disabled tests, usually with comment describing reasons                            |
+| @cli               | Tests to be executed only using cli                                                |
+|                    |                                                                                    |
+| @springboot        | SpringBoot tests                                                                   |
+| @quarkus           | Quarkus tests                                                                      |
+|                    |                                                                                    |
+| @dataindex         | Tests including DataIndex                                                          |
+| @jobsservice       | Tests including Jobs service                                                       |
+| @managementconsole | Tests including Management console                                                 |
+| @infra             | Tests checking KogitoInfra functionality                                           | (should mark also every test which use Kogito infra?)
+| @discovery         | Tests checking service discovery functionality                                     |
+|                    |                                                                                    |
+| @binary            | Tests using Kogito applications built locally and uploaded to OCP as a binary file |
+| @native            | Tests using native build                                                           |
+| @persistence       | Tests verifying persistence capabilities                                           | (or tests running with enabled persistence?)
+| @events            | Tests verifying eventing capabilities                                              | (or tests running with enabled events?)
+| @discovery         | Tests checking service discovery functionality                                     |
```

Comment:
discovery tag is mentioned there twice - on line [1213](https://github.com/kiegroup/kogito-cloud-operator/pull/335/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1213) and here on line 1219